### PR TITLE
Sprint-1 tasks

### PR DESCRIPTION
**Pull request by baneet24**
commit 1:
- Support for multi-env development.
- Added JPA dependency  
- Setup health endpoint to test the APIs
- Made simple GraphQL API call.

commit 2:
- Addressed comments

commit 3:
- Removed unnecessary files and added Junit test
image

### CODE REVIEW

File: src/main/java/com/example/ibeproject/service/GraphQLService.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+
+@Service
+public class GraphQLService {
+    private static final String ROOMS_DATA = "{ listRooms {room_id room_number } }";
+    private static final String APIKEY = "da2-fakeApiId123456";
```
Comment (Zxcy88) [inline src/main/java/com/example/ibeproject/service/GraphQLService.java]: Put it in application.properties
Don't hardcode anything here

File: src/main/java/com/example/ibeproject/service/GraphQLService.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+        public ResponseEntity<String> getRooms(){
+            HttpHeaders httpHeaders = new HttpHeaders();
+            httpHeaders.setContentType(MediaType.APPLICATION_JSON);
+            httpHeaders.set("x-api-key", APIKEY);
+            String requestBody = "{ \"query\": \"" + ROOMS_DATA + "\" }";
+            HttpEntity<String> requestEntity = new HttpEntity<>(requestBody, httpHeaders);
+            try {
+                return restTemplate.exchange("http://localhost:4000/graphql", HttpMethod.POST, requestEntity, String.class);
```
Comment (Zxcy88) [inline src/main/java/com/example/ibeproject/service/GraphQLService.java]: Same, put the endpoint into application.properties

**Review by Sagun15 — CHANGES_REQUESTED** [review]
Address comment

File: src/main/java/com/example/ibeproject/service/GraphQLService.java (hunk 1 above)
Comment (baneet24) [reply src/main/java/com/example/ibeproject/service/GraphQLService.java]: Done

File: src/main/java/com/example/ibeproject/service/GraphQLService.java (hunk 2 above)
Comment (baneet24) [reply src/main/java/com/example/ibeproject/service/GraphQLService.java]: OK, will not hard code

**Review by Sagun15 — APPROVED** [review]

**Pull request merged**